gatherers/rdns.py

```diff
@@ -1,6 +1,7 @@
 import json
 import logging
 import re
+from typing import Generator, List, Pattern
```

Comment:
Will this lock the project to Python 3.7+ as written?

Reply:
It definitely works in 3.6 and I'm pretty sure 3.5 as well, if not earlier versions.